recipes/ipypdf/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set version = "0.0.1" %}
+
+package:
+  name: ipypdf
+  version: {{ version }}
+
+source:
+  url: https://github.com/joelstansbury/ipypdf/archive/0206fb7.tar.gz
+  sha256: ec1e9cfd2db9e45963e2664d86acfdaf7b5c4568e654ceb3c6f60d7a29f65f3d
+
+build:
+  script: {{ PYTHON }} -m pip install -vv . --no-deps
+  noarch: python
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools
+  run:
+    - python >=3.6
+    - pip
+    - ipycanvas
+    - ipycytoscape
+    - ipyevents
+    - ipytree
+    - ipywidgets
+    - jupyterlab >=3.0
+    - numpy <=1.19.3,>=1.13  # Constrained by paddlepaddle
```

Comment:
this is all-but-not-going-to work, long term...

Reply:
I'll try including paddlepaddle as a requirement, but it is on it's own channel (`paddle`). I wasn't sure if the CI would be able to find it

Reply:
As with `layoutparser`, packaging `paddle` (either on this PR or separately) for `conda-forge` is the high road.

As to pulling from a separate channel: it's unlikely to be acceptable. 

As with many of these heavyweight deps, if you control the upstream, making them optional, and gracefully handle being missing, is your best bet for something that will work upstream. The success of conda-forge, as a platform, depends on careful understanding of [high-level packages](https://github.com/conda-forge/conda-forge-pinning-feedstock/) to keep things running over time on multiple platforms, and numpy et. al are on that list.

If you can't handle missing that feature, but know you need a particular version, your out is [`run_constrained`](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#run-constrained)... it won't make it install, but if it _is_ available in the set of channels consulted, it will make sure it it the right version, and then document in your install instructions that both versions are required.